packages/terra-clinical-alert/tests/nightwatch/alert-spec.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-unused-expressions */
+// eslint-disable-next-line import/no-extraneous-dependencies
+const resizeTo = require('terra-toolkit/lib/nightwatch/responsive-helpers').resizeTo;
+
+module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], {
+
+  'Default alert test - Displays a default alert with the provided text': (browser) => {
+    browser
+      .url(`${browser.launchUrl}/#/tests/alert-tests/default`)
+      .expect.element('#defaultAlert').to.be.present.before(1000);
```

Comment:
Nightwatch configuration already waits for 1000 seconds so `.expect.element().to.be.present` will have the same results